pydrumscore/export.py

```diff
@@ -528,6 +529,34 @@ def add_note(chord, notedef):
     with open(save_path, "wb") as f:
         f.write(xml_str)
 
+    zip_file= zipfile.ZipFile("pydrumscore/_exported/" + metadata.workTitle + ".mscz", "w")
+    ########################## Export from: export_files
+    # suggestion: Make a little function and make this locally
+    zip_file.write("pydrumscore/_exported/" + filename, filename, compress_type=zipfile.ZIP_DEFLATED)
+    zip_file.write("pydrumscore/export_files/audiosettings.json", "audiosettings.json", compress_type=zipfile.ZIP_DEFLATED)
+    zip_file.write("pydrumscore/export_files/score_style.mss", "score_style.mss", compress_type=zipfile.ZIP_DEFLATED)
+    zip_file.write("pydrumscore/export_files/viewsettings.json", "viewsettings.json", compress_type=zipfile.ZIP_DEFLATED)
+    zip_file.write("pydrumscore/export_files/Thumbnails/thumbnail.png", "Thumbnails/thumbnail.png", compress_type=zipfile.ZIP_DEFLATED)
+
+    ############################ Create container.xml
+    import xml.etree.ElementTree as ET
+    mytree = ET.parse("pydrumscore/export_files/META-INF/container.xml")
+    myroot= mytree.getroot()
```

Comment:
`mytree` and `myroot` should be renamed to be more descriptive of what they represent.